pkgs/by-name/lz/lz4json/package.nix

```diff
@@ -0,0 +1,36 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  lz4,
+  pkg-config,
+}:
+
+stdenv.mkDerivation {
+
+  pname = "lz4json";
+  version = "2-unstable-2019-12-29";
+  src = fetchFromGitHub {
+    owner = "andikleen";
+    repo = "lz4json";
+    rev = "c44c51005c505de2636cc1e59cde764490de7632";
+    sha256 = "sha256-rLjJ7qy7Tx0htW1VxrfCCqVbC6jNCr9H2vdDAfosxCA=";
+  };
+
+  buildInputs = [
+    lz4
+    pkg-config
+  ];
+
+  installPhase = ''
+    mkdir -p $out/bin
+    install -D lz4jsoncat $out/bin/lz4jsoncat
```

Comment:
I think the `installPhase` can be completely removed.

In the project's readme I see that you can update the C make flags: `make CFLAGS="-I/usr/local/include -O2" LDFLAGS="-L/usr/local/lib"`, therefore, you could just update the make flags accordingly (See examples in `nixpkgs` repo) and remove the custom `installPhase` attribute.